sections/1-introduction.tex

```diff
@@ -1,101 +1,76 @@
+These core packages form the backbone for hundreds of thousands of additional \python packages.
+Of particular relevance to \sunpypkg is the \astropypkg package, which provides core functionality for the analysis of astrophysical data \citep{astropy2018}. 
 
-
-%NumPy 102906 LOC
-%SciPy 139009 LOC
-%matplotlib 98720 LOC
-%pandas 212338 LOC
-%astropy 151825 LOC
-
-%The SunPy Project aims to develop and provide high-quality, maintainable and tested code paired with extensive documentation that follow current best practices in software development. 
-
-%The core library sets a standard and example for other codes to follow.
-
-%A description of the current affiliated packages can be found in Section~\ref{sec:affil_packages}.
-
-\section{Project Organization \& Enhancement Proposals - Steven Christe}
-% edited by Steven C. 23-Apr-2019
+\section{Project Organization \& Enhancement Proposals}
 
 The organization of the \sunpyproj is modeled on the structure of a board-only not-for-profit corporate entity. 
 It consists of an up-to 10 member self-selected board. 
-An executive director, elected by the board, is the lead developer whose responsibilities include leading the developer community, providing user support, developing and maintaining the core package, as well as supporting the development of affiliated packages. 
-The lead developer is supported by a deputy as well as other volunteers from the developer community. 
+An executive director, elected by the board, leads the development of \sunpy core package, provides user support, and supports the development of affiliated packages. 
+As such, the executive director is also the \sunpy lead developer.
+A deputy lead developer and release manager, as well as other volunteers from the developer community, support the lead developer.
 Board members serve two year terms while the lead developer serves one year terms. 
 
-The \sunpyproj is formally defined through \sunpy Enhancement Proposals (SEPs) which are modeled after the Python Enhancement Proposal process\footnote{\url{https://www.python.org/dev/peps/}}. 
-The first two SEPs (SEP-0001 and SEP-0002) define themselves as well as the \sunpy organization. 
-They are version controlled and publicly available\footnote{\url{https://github.com/sunpy/sunpy-SEP}}. 
-SEPs are both used to define the project as well as requirements on the the \sunpypkg package. 
+The \sunpyproj is formally defined through \sunpy Enhancement Proposals (SEPs) which are modeled after the Python Enhancement Proposal process\footnote{\url{https://www.python.org/dev/peps/}}.
+SEPs are version controlled and publicly available\footnote{\url{https://github.com/sunpy/sunpy-SEP}}. The first two SEPs (SEP-0001 and SEP-0002) define themselves as well as the \sunpy organization. 
```

Comment:
Should we take this opportunity to follow astropy and put our SEPs on Zenodo so we can correctly cross reference them here? x/ref sunpy/sunpy-SEP#33

Reply:
How would that work? Do a release every time we add a new one? or is each one have it's own DOI?

Reply:
We should move this discussion to the SEP repo, but each one would have it's own DOI. I assume when we update one there would be a release process as with accepting a new one.